websites/C/ClarityClan/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "Leonardo Jacon#0001",
+      "id": "600148454668369930"
+    },
+    "url": "clarityclan.team",
+    "description": {
+      "en": "Clarity Clan is a Minecraft Clan, growing in the community of MCC."
+    },
+    "service": "Clarity Clan",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/LGdKN10.png",
+    "thumbnail": "https://i.imgur.com/LGdKN10.png",
```

Comment:
The thumbnail must not be the same as the logo, take a screenshot of the index page and use that instead.